controller/databaseManager.py

```diff
@@ -76,4 +76,4 @@ def removeReader(self, id):
         self.modelReaders.select()
 
 
-dbManager = DatabaseManager()
+dbManager = DatabaseManager()
```

Comment:
Nie wiem czy konieczne było usunięcie tej pustej linii

Reply:
Ok to niedopatrzenie. Poprawiam